ci/feature-soundness-release.sh

```diff
@@ -18,15 +18,3 @@ cargo clippy --release --no-default-features --features default,ssr -- --deny=wa
 cargo clippy --release --no-default-features --features csr,default,ssr -- --deny=warnings
 cargo clippy --release --no-default-features --features hydration,ssr -- --deny=warnings
 cargo clippy --release --no-default-features --features default,hydration,ssr -- --deny=warnings
-cargo clippy --release --no-default-features --features tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features csr,tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features default,tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features csr,default,tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features hydration,tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features default,hydration,tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features ssr,tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features csr,ssr,tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features default,ssr,tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features csr,default,ssr,tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features hydration,ssr,tokio -- --deny=warnings
-cargo clippy --release --no-default-features --features default,hydration,ssr,tokio -- --deny=warnings
```

Comment:
It is removed completely and does not present in prokio as well. As previously mentioned, the tokio feature is not very useful. I expect the correct situation to stay years before it changes. So I guess it might be better to remove tokio feature and read it when we need it.